src/middlewared/middlewared/plugins/acme_protocol_/issue_cert.py

```diff
@@ -97,6 +97,18 @@ def issue_certificate(self, job, progress, data, csr_data):
                     )
                 except errors.TimeoutError:
                     raise CallError('Certificate request for final order timed out')
+                except errors.ValidationError as e:
+                    msg = ''
+                    for authzr in e.failed_authzrs:
+                       msg += f'\n Authorization for identifier {authzr.body.identifier} failed.'
```

Comment:
Why a space after a new line?

Reply:
Why? Me being stupid and not actually reading it like I should. >.<
Thanks for the headsup. Should be all fixed up now :)